File: exercise/income.py
def make_benefit(people, ticket_price):
    income=people*ticket_price
    outcome=180+(people*0.04)
    total_income=income-outcome
    return total_income

ticket=5
people_count=120
max=0

for i in range(0,50):
    money=make_benefit(people_count, ticket)
    if money > max:
        max=money
        best_ticket_price=ticket

    ticket=ticket-0.1
    people_count=people_count+15

print(best_ticket_price, ":", max)

File: Hello/src/Income.java
public class Income {
	public static void main(String[] args) {
		float ticket = 5;
		int people = 120;
		float max = 0;
		float best_ticket = 0;
		
		for(int i=0;i<50;i++) {
			
			float income = people * ticket;
			float outcome = (float) (180 + (people * 0.04));
			float total_income = income - outcome;
			
			if (total_income > max){
				max = total_income;
				best_ticket = ticket;
			}
			
			System.out.println(ticket + " " + total_income);
			ticket -= 0.1;
			people += 15;
			
			
		}
		System.out.println(best_ticket + " " + max);
	}
	
}
